### ISSUE TITLE
LSEP Characters

### ISSUE THREAD

**Issue by Samueldriessen** [distilagency/koval#4]
LSEP characters have been popping up on recent builds. These don't show up on Mac but are shown on Windows computers. They also display on Android as an unloaded character. Unfortunately in previous instances the client has been the first to notice this issue, as they are possibly the first user to see the website on Windows.

We need to figure out where in the process these are being loaded, somewhere in the way we are adding/copying content onto the website. From there we need to figure out a way to stop this from occurring.

![lsep-windows](https://user-images.githubusercontent.com/26733345/53848932-2613a280-3ff1-11e9-8a7f-66f6a230202c.PNG)
![lsep-android](https://user-images.githubusercontent.com/26733345/53848933-2613a280-3ff1-11e9-852c-5452e0f61838.png)

**Comment by samlogan** [comment]
Looks like they might be coming across from the WP WYSIWYG editor, will investigate @Samueldriessen

**Comment by Samueldriessen** [comment]
Thanks Sam. I have removed this from Capital Partners in preparation for site launch.

**Comment by Samueldriessen** [comment]
Hey @samlogan possible update on this - it could be caused when copying content from sketch, not sure of this though.

**Comment by samlogan** [comment]
@samueldriessen yeah that’s what I have traced it back to. Making some updates to Koval next week so will try and address it then

**Comment by Samueldriessen** [comment]
Hi @samlogan , Have we had any movement on this? Mark is seeing this on GAPerry, may be worth finding preventative or post-fix methods for it.

**Comment by samlogan** [comment]
@Samueldriessen I believe @chungeric has been looking into it but it's a hard one to fix as it appears to be an issue with the character encoding at WordPress's end, possibly from the database schema allowing in UTF characters when copying and pasting into their WYSIWYG editor from  Sketch.

**Comment by markmollart** [comment]
The solution I have found so far is to copy a red dot from the REST API: 
<img width="391" alt="Screen Shot 2019-05-06 at 11 00 53 am" src="https://user-images.githubusercontent.com/5590681/57205122-d4869480-6fee-11e9-90fd-4b82e5e26594.png">
paste this into a database find and replace tool like https://interconnectit.com/products/search-and-replace-for-wordpress-databases/ and replace with nothing.

**Comment by markmollart** [comment]
I was able to get the character removed from the REST Json feed with this update (please note this is specific to GA Perry)

https://github.com/birdbrain/gaperry-wp/commit/40e21d7c56e2af58a4232f1f6ffc85c6faa53f75

**Comment by samlogan** [comment]
@markmollart that’s great
